### ISSUE TITLE
Permission denial when restoring backup 

### ISSUE THREAD

**Issue by MrDfour** [null2264/yokai#111]
### Steps to reproduce

Migrated  backup from another device with whole folder
Restore successfully yet this popped after the message of success.
![Screenshot_2024-06-26-18-28-31-732_com mi android globallauncher-edit](https://github.com/null2264/yokai/assets/103494853/3dc214d1-68e1-41a2-8c1f-f4d96f61c9c2)
Not gonna lie, I don't see anything missing, the chapters seem to be correct and up to date but still wanted to report just in case it is something noteworthy 

### Expected behavior

The 3rrir shouldn't pop? 

### Actual behavior

A weird error pops up 

### Yōkai version

1.8.4.1

### Android version

14

### Device

Poco x6 Pro 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, or a request for an extension, I should be contacting the extensions repository's maintainer/support for help.
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Mihon](https://github.com/mihonapp/mihon)
- [X] I have tried the [troubleshooting guide](https://mihon.app/help/).
- [X] I have updated the app to version **[1.8.4.1](https://github.com/null2264/yokai/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by null2264** [comment]
> Restore successfully yet this popped after the message of success

Both failed and success notification appeared?

**Comment by MrDfour** [comment]
> > Restore successfully yet this popped after the message of success
> 
> Both failed and success notification appeared?

Yep, and as far as I can tell everything seems to be right. To give more context, this was my main device but I broke it so used my secondary to read and stuff. When I repaired my main I did a manual backup of secondary and migrated to main. Almost the same Manga, just one migrated to another source and that's it as far as I recall.
But then again, migration went successfully and with no issues. Just a minor glitch that made the last chapter appear beneath the last chapter when main device broke, it solved with a global update
